AffinityPanel.c

```diff
@@ -400,7 +400,7 @@ Panel* AffinityPanel_new(Machine* host, const Affinity* affinity, int* width) {
 
       char number[16];
       xSnprintf(number, 9, "CPU %d", Settings_cpuId(host->settings, i));
-      unsigned cpu_width = 4 + strlen(number);
+      unsigned cpu_width = 4 + (unsigned)strlen(number);
```

Comment:
While `unsigned` is a valid alias for `unsigned int` in C, it would be preferable to use `unsigned int` to be more explicit about the referenced type. Cf. my recent cleanup PR #1675 …